.travis.yml

```diff
@@ -1,65 +0,0 @@
-language: python
-
-matrix:
-  include:
-    - python: 3.6
-      env: TOXENV=py36
-    - python: 3.7
-      env: TOXENV=py37
-    - python: 3.8
-      env: TOXENV=py38
-    - python: 3.6
-      env: TOXENV=style
-      install: true
-
-os:
-  - linux
-
-sudo: false
-dist: bionic
-
-env:
-  global:
-    - INSTALL_DIR="$(pwd)/install"
-    - LD_LIBRARY_PATH="${INSTALL_DIR}/lib:${INSTALL_DIR}/lib64"
-    - DYLD_LIBRARY_PATH="${INSTALL_DIR}/lib:${INSTALL_DIR}/lib64"
-
-addons:
-  apt:
-    sources:
-      - ubuntu-toolchain-r-test
-    packages:
-      - liblapack-dev
-
-install:
-    - git clone --branch master --depth 1 https://github.com/equinor/ert
-    - pushd ert
-    - source .libres_version
-    - popd
-    - git clone --branch $LIBRES_VERSION --depth 1 https://github.com/equinor/libres
-    - pushd libres
-    - pip install -r requirements.txt --prefix=$INSTALL_DIR
-    - source .libecl_version
-    - popd
-    - git clone --branch $LIBECL_VERSION --depth 1 https://github.com/equinor/libecl
-    - bash ert/.build_install.sh libecl
-    - bash ert/.build_install.sh libres
-    - export PYTHONPATH=$INSTALL_DIR/lib/python$TRAVIS_PYTHON_VERSION/dist-packages:$PYTHONPATH
-    - export PYTHONPATH=$INSTALL_DIR/lib/python$TRAVIS_PYTHON_VERSION/site-packages:$PYTHONPATH
-    - PATH=$INSTALL_DIR/bin:$PATH
-    - pip install ert/ --prefix=$INSTALL_DIR
-
-script:
-  - pip install tox
-  - tox
-
-deploy:
```

Comment:
I know I said you could delete the travis file, but until I can set up deploy on GitHub actions, we need this part.